install_config/topics/configuring_for_vsphere_storage.adoc

```diff
@@ -41,17 +41,57 @@ storage provider.
 Before creating PVs using vSphere, ensure your
 {product-title} cluster meets the following requirements:
 
-* {product-title} must first be configured for vSphere Cloud Provider
+* {product-title} must first be
```

Comment:
No xrefs in modules.

Reply:
So what would be the best here ? 
( sorry new to adoc formatting )